lib/utils.js

```diff
@@ -160,7 +146,8 @@ exports.cheerio = function(html, options) {
 };
 
 exports.normalizeLineEndings = function(text) {
-  return text.replace(/\r\n/g, '\n').replace(/\n/g, '\r\n');
+  // NOTE this consider multiple newlines the same as a single newline
+  return text.replace(/\r\n/g, '\n').replace(/\n/g, '\r\n').replace(/\r\n\r\n/g, '\r\n');
```

Comment:
There are differences in newlines in the way mensch writes CSS and juice wrote CSS (i talk about @media and @font-face rules). Instead of altering all of the ".out" files I decided that a CSS with one more or one less empty line can be considered the same for the tests.

What I tried to do is to do the strict changes to the "expected" out files, so it is easier to see the *real* differences between the two parsers.

Of course this replace can be changed if we recreate the out files as mensch wants them.